bool valid
bool grandmaster_present
uint8[8] grandmaster_id
int64 grandmaster_offset
int64 path_delay
int16 steps_removed
